app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -80,170 +81,173 @@ export const ContentForm: React.FC = () => {
   };
 
   return (
-    <styled.ContentForm>
-      <Area>
-        <Row>
-          <Button variant={ButtonVariant.action} onClick={() => navigate('/contents')}>
-            <Row>
-              <img
-                style={{ marginRight: '0.5em' }}
-                alt="back"
-                src={process.env.PUBLIC_URL + '/assets/back_arrow.svg'}
-              />
-              <p>Back to List View</p>
-            </Row>
-          </Button>
-        </Row>
-        <FormikForm
-          onSubmit={handleSubmit}
-          validationSchema={ContentFormSchema}
-          initialValues={content}
-        >
-          {(props) => (
-            <Col>
-              <Row alignItems="flex-start">
-                <Col flex="1 1 auto">
-                  <FormikText
-                    name="headline"
-                    required
-                    label="Headline"
-                    value={props.values.headline}
-                  />
-                  <Row>
-                    <Col>
-                      <FormikSelect
-                        name="dataSourceId"
-                        label="Source"
-                        value={
-                          dataSourceOptions.find((mt) => mt.value === props.values.dataSourceId) ??
-                          ''
-                        }
-                        onChange={(e: any) => {
-                          props.setFieldValue('dataSourceId', e.value);
-                          props.setFieldValue('source', e.label);
-                        }}
-                        width={FieldSize.Medium}
-                        options={dataSourceOptions}
-                        required={!props.values.source}
-                        isDisabled={!!props.values.otherSource}
-                      />
-                      <FormikHidden name="source" />
-                    </Col>
-                    <Col>
-                      <FormikText
-                        width={FieldSize.Medium}
-                        name="otherSource"
-                        label="Other Source"
-                        onChange={(e) => {
-                          const value = e.currentTarget.value;
-                          props.setFieldValue('otherSource', value);
-                          props.setFieldValue('source', value);
-                          if (!!value) {
-                            props.setFieldValue('dataSourceId', undefined);
+    <FormPage height="fit-content">
```

Comment:
This isn't ideal.  Generally it's preferred to always have the styled component as the root object returned.  Either the `styled.ContentForm` should inhert from `FormPage` or it should wrap it.

Reply:
Also trying to understand why your auto formatting is messing with the whole file.  It's impossible to see what has changed.

Reply:
In this case it was because of the new indentation due to adding a new parent